doc/avifdec.1.md

```diff
@@ -31,13 +31,11 @@ Output format can be either JPEG, PNG or YUV4MPEG2 (Y4M).
 :   Show the version number.
 
 **-j**, **\--jobs** _J_
-:   Number of jobs (worker threads).
-    1 or less means single-threaded.
-    Default is 1.
-    Use **all** to use all available cores.
+:   Number of jobs (worker threads). Use "all" to potentially use as many cores as possible (default: all).
```

Comment:
The commit message says "Don't impose any line length limit in the markdown files to minimize work." This is fine by me. If there is a standard markdown file formatter, we can also use that.